### ISSUE TITLE
Login and Registration not working

### ISSUE THREAD

**Issue by tonypersonal** [nopSolutions/nopCommerce#4125]
nopCommerce version: 4.20

Steps to reproduce the problem: 
I installed in my local machine. I have give the local server name and database name, now all the tables and procedures are created.

Homepage is opening after the I click on login and put my username and password, it gives the following error.

![image](https://user-images.githubusercontent.com/52854612/67296224-cc230700-f505-11e9-90ad-bc0312d9d917.png)

**Comment by DmitriyKulagin** [comment]
Hi, @tonypersonal 
I just tried to reproduce the problem and it works well. Did you make any changes to the source code? Or installed third-party plugins?

**Comment by nguyenhthai** [comment]
I also encountered this quite a few recently on my dev environment. Don't know what exactly the cause but it usually happened when the server is starting from cold and the AppPool is init as my AppPool is configured to suspend/terminate after a certain timeout. When that happened, I went back to the login page, tried the same credential and it worked normally.  In the System/Log area, there is an entry for this with the log just has the short message of "**Error 400. Bad request**" and no full message at all.

No third-party plugin install and I only have the plugin that I have been working on installed at that moment. Also, no core source code had been modified. Will try to get a full backend log if possible.
